spec/features/variable_drafts/forms/validation_spec.rb

```diff
@@ -89,8 +89,8 @@
           expect(page).to have_field('Name', with: 'Test Var Short Name')
           expect(page).to have_field('Definition', with: 'Definition of test variable')
           expect(page).to have_field('Long Name', with: 'Test Var Long Long Name')
-          expect(page).to have_field('Scale', with: '2.0')
-          expect(page).to have_field('Offset', with: '5.0')
+          expect(page).to have_field('Scale', with: '2')
+          expect(page).to have_field('Offset', with: '5')
```

Comment:
I recall you mentioned that this may need to be changed back. Is this that the case, or should this be ok as is?

Reply:
They have been changed in 1267 already